### PR TITLE
Make lazy installable via NPM

### PR DESCRIPTION
**Pull request by JamieLivingstone**

### CODE REVIEW

**Comment by MikeDigitize** [comment]
@JamieLivingstone Can you provide a bit of extra detail as to why you've replaced Object spread with Object assign please?

File: __tests__/lazy-scroll.test.js [hunk 1]
```diff
@@ -310,4 +310,35 @@ describe('LazyScroll class tests', function() {
...
+			window.scrollTo(
+				lazyImage.imagePosition.top,
+				lazyImage2.imagePosition.top
+			);
+
+			setTimeout(function() {
+				expect(lazyImage2.resolved).toBe(false);
+			}, 1000);
+
+			done();
```
Comment (nardan) [inline __tests__/lazy-scroll.test.js]: Shouldn't this `done()` be in the timeout?

**Comment by nardan** [comment]
Associated issue #19

File: __tests__/lazy-scroll.test.js (hunk 1 above)
Comment (MikeDigitize) [reply __tests__/lazy-scroll.test.js]: Good spot! Would appear so!

**Pull request merged**